src/Context/Settings/index.jsx

```diff
@@ -19,11 +21,12 @@ function SettingsProvider({ children }) {
   useEffect(() => {
     const storedSettings = localStorage.getItem('settings');
     if (storedSettings) {
-      const { displayCount, showCompleted, sortField, itemsToShow } = JSON.parse(storedSettings);
+      const { displayCount, showCompleted, sortField, firstRender } = JSON.parse(storedSettings);
       setDisplayCount(displayCount);
       setShowComplete(showCompleted);
       setSort(sortField);
-      setItemsToShow(itemsToShow);
+      setFirstRender(firstRender);
```

Comment:
When you are destructuring items from your parsed local storage, make sure the property names match; you are setting `sort` and `showComplete` to undefined here because `showCompleted` and `sortField` do not exist.

Also, we do not want to store/load data for `firstRender`. It will always be true when we load the page, then will be set to false when the other useEffect is run on page load.